package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "dependencies": {
+    "@google-ai/generativelanguage": "^1.1.0",
+    "google-auth-library": "^9.3.0"
```

Comment:
In general what's the best practice for pinning dependencies?

Reply:
we haven't pinned any dependencies in AIConfig besides ones that may have broken with latest. We could probably pin dependencies now that AIConfig is a bit more mature than when we were first building